tuf/pyca_crypto_keys.py

```diff
@@ -309,15 +309,11 @@ def create_rsa_signature(private_key, data):
   # has variable size and can be an empty string.
   if len(private_key):
     
-    # pyca/cryptography's expected exceptions when generating RSA key object:
-    # "ValueError/IndexError/TypeError:  When the given key cannot be parsed
-    # (possibly because the passphrase is wrong)."
-    # A passphrase or password is not used to 'private_key', since it should
-    # not be encrypted.
+    # Generate an RSSA-PSS signature.  Raise 'tuf.CryptoError' for any of the
```

Comment:
While this addresses [Albert's initial comment on PR #289](https://github.com/theupdateframework/tuf/pull/289/files#r48976488), it looks like this does not provide a clear explanation of what is happening. I suggest something along the lines of Albert's edited comment: "Converts a private key from PEM format to a pyca/cryptography object".